HelloWorldService/src/main/resources/application.conf

```diff
@@ -50,6 +50,11 @@ helloworld {
   weatherTemplate: "Hello %s, the weather is %s!"
 }
 
+weather {
+  darkSkyApiKey: ""
+  darkSkyApiKey: ${?DARK_SKY_API_KEY}
+}
```

Comment:
Do we need line 54?  What happens if it's missing?

Reply:
If the env var is not set and line 54 is missing, the application gets an Exception on startup saying that darkSkyApiKey is not set.

Reply:
I vote we keep line 54.